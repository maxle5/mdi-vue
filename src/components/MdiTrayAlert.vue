
<template>
  <svg :width="size" :height="size" viewBox="0 0 24 24">
      <path d="M2 12H4V17H20V12H22V17A2 2 0 0 1 20 19H4A2 2 0 0 1 2 17M13 12H11V14H13M13 4H11V10H13Z" style="fill: currentColor" />
  </svg>
</template>

<script lang="ts">
  import { defineComponent } from 'vue';

  export default defineComponent({
      name: "MdiTrayAlert",
      props: {
          size: { 
              type: Number, 
              default: 24 
          }
      },
  })
</script>
