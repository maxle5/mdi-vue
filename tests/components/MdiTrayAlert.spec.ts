
import { mount } from "@vue/test-utils";
import { expect, test } from "vitest";
import MdiTrayAlert from "../../src/components/MdiTrayAlert.vue";

test("MdiTrayAlert snapshot", () => {
  const wrapper = mount(MdiTrayAlert, {});
  expect(wrapper.html()).toMatchSnapshot();
});
