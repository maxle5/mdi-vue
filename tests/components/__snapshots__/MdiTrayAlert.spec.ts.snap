// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`MdiTrayAlert snapshot 1`] = `
"<svg width="24" height="24" viewBox="0 0 24 24">
  <path d="M2 12H4V17H20V12H22V17A2 2 0 0 1 20 19H4A2 2 0 0 1 2 17M13 12H11V14H13M13 4H11V10H13Z" style="fill: currentColor;"></path>
</svg>"
`;
